project/cmake/modules/FindPython.cmake

```diff
@@ -5,7 +5,7 @@
 # PYTHON_INCLUDE_DIRS - the python include directory
 # PYTHON_LIBRARIES - The python libraries
 
-if(PKG_CONFIG_FOUND AND NOT CMAKE_CROSSCOMPILING)
```

Comment:
@wsnipex: Are you familiar with the constrains of this module? (going back in the history i could see that you added the NOT CMAKE_CROSSCOMPILING here). according to the build log, i'd also have to exclude linux here now. which basically means we only use pkgconfig on osx.